gradio/external.py

```diff
@@ -418,7 +418,7 @@ def from_spaces(
 
 
 def from_spaces_blocks(space: str, hf_token: str | None) -> Blocks:
-    client = Client(space, hf_token=hf_token, download_files=False)
+    client = Client(space, hf_token=hf_token, upload_files=False, download_files=False)
```

Comment:
Nope it works, because files are automatically preprocessed and converted to a FileData by the loaded Gradio app before they are sent to the remote Space

Reply:
I got an error with this code (no output image when uploading my own file) 

```python
gr.load("spaces/gradio/image_mod").launch()
```

Maybe my environment was messed up - can you repro?

Reply:
This should now be fixed